### PR TITLE
(PDB-841) Retire the legacy storeconfigs tool

### PR DESCRIPTION
**Pull request by ajroetker**
This commit removes support for the legacy storeconfigs export tool as
we no longer support Puppet <4 in PuppetDB.

### CODE REVIEW

File: acceptance/tests/storeconfigs/basic_collection.rb [hunk 1]
```diff
@@ -1,47 +0,0 @@
-test_name "general collection should get all exported resources except the host's" do
```
Comment (kbarber) [inline acceptance/tests/storeconfigs/basic_collection.rb]: Why is this test being removed?

Comment (ajroetker) [inline acceptance/tests/storeconfigs/basic_collection.rb]: I mistakenly removed the whole directory `storeconfigs`

File: acceptance/tests/storeconfigs/collections_with_queries.rb [hunk 2]
```diff
@@ -1,120 +0,0 @@
-test_name "collections with queries" do
```
Comment (kbarber) [inline acceptance/tests/storeconfigs/collections_with_queries.rb]: Same again, not sure why this is being removed? So the ticket was to remove the _legacy_ storeconfigs export tool, these tests are still relevant without the tool right? That is, we still have storeconfigs without the legacy export mechanism don't we?

File: acceptance/tests/storeconfigs/legacy_storeconfigs_import_export.rb [hunk 3]
```diff
@@ -1,75 +0,0 @@
-puppetdb_query_url = "http://localhost:8080/pdb/query"
```
Comment (kbarber) [inline acceptance/tests/storeconfigs/legacy_storeconfigs_import_export.rb]: Now this test makes sense to remove, I'm not sure anything else in this directory does though ...

File: acceptance/tests/storeconfigs/collections_with_queries.rb (hunk 2 above)
Comment (ajroetker) [inline acceptance/tests/storeconfigs/collections_with_queries.rb]: Yep, responded on the other comment, removed the whole directory by mistake.

**Comment by kbarber** [comment]
@ajroetker looking good, I think we just have removed too many tests :-).

**Comment by ajroetker** [comment]
transient travis failure ^

**Pull request closed without merging**